integration-tests/xmlsecurity/src/main/resources/application.properties

```diff
@@ -0,0 +1,17 @@
+## ---------------------------------------------------------------------------
+## Licensed to the Apache Software Foundation (ASF) under one or more
+## contributor license agreements.  See the NOTICE file distributed with
+## this work for additional information regarding copyright ownership.
+## The ASF licenses this file to You under the Apache License, Version 2.0
+## (the "License"); you may not use this file except in compliance with
+## the License.  You may obtain a copy of the License at
+##
+##      http://www.apache.org/licenses/LICENSE-2.0
+##
+## Unless required by applicable law or agreed to in writing, software
+## distributed under the License is distributed on an "AS IS" BASIS,
+## WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+## See the License for the specific language governing permissions and
+## limitations under the License.
+## ---------------------------------------------------------------------------
+quarkus.camel.native.resources.include-patterns=*.xsl
```

Comment:
Could we use quarkus.native.resources.includes directly here ?

Reply:
Yes, I actually forgot it had been added to Quarkus core (I guess this can be closed now https://github.com/apache/camel-quarkus/issues/988).

Maybe we should update all of the tests to use it and deprecate the old config properties? I hate it when we have multiple ways of doing the same thing.

Reply:
sure

Reply:
Yeah, it took some time, but now both includes and excludes are supported by Quarkus.